ompi/mpi/man/man3/MPI_File_get_size.md

```diff
@@ -0,0 +1,78 @@
+# Name
+
+`MPI_File_get_size` - Returns the current size of the file.
+
+# Syntax
+
+## C Syntax
+
+```c
+#include <mpi.h>
+
+int MPI_File_get_size(MPI_File fh, MPI_Offset *size)
+```
+
+## Fortran Syntax (See Fortran 77 Notes)
+
+```fortran
+USE MPI
+! or the older form: INCLUDE 'mpif.h'
+
+MPI_FILE_GET_SIZE(FH, SIZE, IERROR)
+    INTEGER    FH, ERROR
+
+    INTEGER(KIND=MPI_OFFSET_KIND)    SIZE
+```
+
+## Fortran 2008 Syntax
+
+```fortran
+USE mpi_f08
+
+MPI_File_get_size(fh, size, ierror)
+    TYPE(MPI_File), INTENT(IN) :: fh
+    INTEGER(KIND=MPI_OFFSET_KIND), INTENT(OUT) :: size
+    INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+```
+
+
+# Input Parameters
+
+* `fh` : File handle (handle).
+* `size` : Size of the file in bytes (integer).
+
+# Output Parameter
+
+* `IERROR` : Fortran only: Error status (integer).
+
+# Description
+
+`MPI_File_get_`size`` returns, in ``size`` , the current `size` in bytes of the
```

Comment:
```suggestion
`MPI_File_get_size` returns, in `size` , the current `size` in bytes of the
```

Might want to have the Python check for double quoting like this, to make sure you don't end up with 2 quotes in a row.  That's a sign that you ended up double quoting something.